### DISCUSSION TITLE
（灌水）获取文章失败怎么办啊

### DISCUSSION THREAD

**Discussion by HLXS-HLXS** [share121/inter-knot#2978] [其他]
![Screenshot_20240819_093331_mark via](https://github.com/user-attachments/assets/a0ddda43-7d17-4bca-8af0-d81ed44aa818)
文章获取不了啊，挂梯子也没用

**Comment by nostalgiatan** [comment]
如果你用的是绳网.top获取文章失败的话，你或许可以用这个链接https://share121.github.io/inter-knot/
如果显示1.8就数据删一下。

**Reply by HLXS-HLXS** [reply]
谢谢，我试试

**Reply by HLXS-HLXS** [reply]
失败

**Comment by youyiMC** [comment]
可以清除一下缓存试试<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/bb90b6b0719fdc45f6375db9eb2b682e_1030720107003730170.gif" alt="维尔薇-魔术" width="75" >

**Reply by HLXS-HLXS** [reply]
试了，失败

**Comment by HLXS-HLXS** [comment]
还有其他办法吗
![1723407120689](https://github.com/user-attachments/assets/bcd6bd7a-257d-40e7-8f2c-da7d76fbcbcf)
